src/content/docs/fr/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,997 @@
+
+Vous trouverez [des instructions complètes et étape par étape pour supprimer l'ancienne intégration d'image](#supprimé-astrojsimage) dans notre guide Images.
+
+La migration vers `astro:assets` apportera également de nouvelles options et fonctionnalités d'image que vous souhaiterez peut-être désormais utiliser. Veuillez consulter l'intégralité des [Conseils de mise à niveau d'image v3.0](#mettre-à-niveau-les-images-vers-la-v3) pour plus de détails !
+
+```js del={3,7}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import image from '@astrojs/image';
+
+export default defineConfig({
+  integrations: [
+    image(),
+  ]
+})
+```
+
+### Supprimé : composant `<Markdown />`
+
+Dans Astro v1.x, Astro a rendu obsolète le composant `<Markdown />` et l'a déplacé vers un paquet externe.
+
+Astro v3.0 supprime complètement le paquet `@astrojs/markdown-component`. Le composant `<Markdown />` d'Astro ne fonctionnera plus dans votre projet.
+
+#### Que devrais-je faire ?
+
+Supprimez toutes les instances de `@astrojs/markdown-component`.
+
+```astro del={2} title="src/components/MyAstroComponent.astro"
+---
+import Markdown from '@astrojs/markdown-component';
+---
+```
+
+Pour continuer à utiliser un composant `<Markdown />` similaire dans votre code, envisagez d'utiliser les [intégrations communautaires](https://astro.build/integrations/) comme [`astro-remote`](https://github.com/natemoo-re/astro-remote). Assurez-vous de mettre à jour vos importations et attributs de composants `<Markdown />` si nécessaire, conformément à la propre documentation de l'intégration.
+
+Sinon, supprimez toutes les références d'importation du composant `<Markdown />` d'Astro et du composant lui-même dans vos fichiers `.astro`. Vous devrez réécrire votre contenu directement au format HTML ou [importer Markdown](/fr/guides/markdown-content/#importer-du-contenu-markdown) à partir d'un fichier `.md`.
+
+### Supprimé : API obsolètes de la v1.x
+
+Dans Astro v1.x, Astro a rendu obsolète nos paramètres de configuration d'origine ainsi que la prise en charge de `<style global>` et `<script hoist>`. Cependant, ceux-ci étaient toujours pris en charge pour des raisons de rétrocompatibilité.
+
+Astro v3.0 supprime entièrement ces API obsolètes. Les [paramètres de configuration](/fr/reference/configuration-reference/) officiellement pris en charge et les syntaxes modernes `<style is:global>` et `<script>` doivent être utilisées à la place.
+
+#### Que devrais-je faire ?
+
+Si vous continuez à utiliser les API v1.x, utilisez plutôt les nouvelles API pour chaque fonctionnalité :
+
+- Options de configuration obsolètes : voir [le guide de migration 0.26](/fr/guides/upgrade-to/v1/#nouvelle-api-de-configuration)
+- Types d'attributs de script/style obsolètes : voir [le guide de migration 0.26](/fr/guides/upgrade-to/v1/#nouveau-comportement-de-script-par-défaut)
+
+### Supprimé : Shims partiels pour les API Web dans le code serveur
```

Comment:
In French, I don't think we have a word for `shims` in this context (computer science). In general context, `shims` could be translated with `cales`. However, in this context (computer science), it seems weird so after checking other French websites, I chose to leave it as `shims` like them.
If you think it should be translated, feel free to leave a suggestion.

Reply:
leave this word 👍🏼